2021-05-09 12:45:59,902 INFO    MainThread:14293 [wandb_setup.py:_flush():69] Unhandled environment var: WANDB_LOG_MODEL
2021-05-09 12:45:59,902 INFO    MainThread:14293 [wandb_setup.py:_flush():69] Unhandled environment var: WANDB_WATCH
2021-05-09 12:45:59,902 INFO    MainThread:14293 [wandb_setup.py:_flush():69] setting env: {'silent': 'true'}
2021-05-09 12:45:59,902 INFO    MainThread:14293 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-09 12:45:59,903 INFO    MainThread:14293 [wandb_setup.py:_flush():69] [34m[1mwandb[0m: Currently logged in as: [33mdlrgy22[0m (use `wandb login --relogin` to force relogin)
2021-05-09 12:45:59,903 INFO    MainThread:14293 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-09 12:45:59,903 INFO    MainThread:14293 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/mycode/code/wandb/run-20210509_124559-16yz9f39/logs/debug.log
2021-05-09 12:45:59,903 INFO    MainThread:14293 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/mycode/code/wandb/run-20210509_124559-16yz9f39/logs/debug-internal.log
2021-05-09 12:45:59,903 INFO    MainThread:14293 [wandb_init.py:init():369] calling init triggers
2021-05-09 12:45:59,903 INFO    MainThread:14293 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-09 12:45:59,903 INFO    MainThread:14293 [wandb_init.py:init():418] starting backend
2021-05-09 12:45:59,903 INFO    MainThread:14293 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-09 12:45:59,969 INFO    MainThread:14293 [backend.py:ensure_launched():123] starting backend process...
2021-05-09 12:46:00,026 INFO    MainThread:14293 [backend.py:ensure_launched():128] started backend process with pid: 14440
2021-05-09 12:46:00,028 INFO    MainThread:14293 [wandb_init.py:init():423] backend started and connected
2021-05-09 12:46:00,030 INFO    MainThread:14293 [wandb_init.py:init():465] updated telemetry
2021-05-09 12:46:00,031 INFO    MainThread:14293 [wandb_init.py:init():484] communicating current version
2021-05-09 12:46:00,852 INFO    MainThread:14293 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.30 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-09 12:46:00,852 INFO    MainThread:14293 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-09 12:46:01,066 INFO    MainThread:14293 [wandb_init.py:init():522] starting run threads in backend
2021-05-09 12:46:03,820 INFO    MainThread:14293 [wandb_run.py:_console_start():1554] atexit reg
2021-05-09 12:46:03,821 INFO    MainThread:14293 [wandb_run.py:_redirect():1401] redirect: SettingsConsole.REDIRECT
2021-05-09 12:46:03,821 INFO    MainThread:14293 [wandb_run.py:_redirect():1406] Redirecting console.
2021-05-09 12:46:03,823 INFO    MainThread:14293 [wandb_run.py:_redirect():1468] Redirects installed.
2021-05-09 12:46:03,823 INFO    MainThread:14293 [wandb_init.py:init():546] run started, returning control to user process
2021-05-09 12:46:37,023 INFO    MainThread:14293 [wandb_run.py:_atexit_cleanup():1524] got exitcode: 255
2021-05-09 12:46:37,023 INFO    MainThread:14293 [wandb_run.py:_restore():1496] restore
2021-05-09 12:46:39,230 INFO    MainThread:14293 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1018
  total_bytes: 1018
}

2021-05-09 12:46:40,093 INFO    MainThread:14293 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1018
  total_bytes: 4133
}

2021-05-09 12:46:40,475 INFO    MainThread:14293 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3769
  total_bytes: 4133
}

2021-05-09 12:46:40,577 INFO    MainThread:14293 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3769
  total_bytes: 4133
}

2021-05-09 12:46:40,679 INFO    MainThread:14293 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4133
  total_bytes: 4133
}

2021-05-09 12:46:40,780 INFO    MainThread:14293 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4133
  total_bytes: 4133
}

2021-05-09 12:46:40,882 INFO    MainThread:14293 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4133
  total_bytes: 4133
}

2021-05-09 12:46:40,983 INFO    MainThread:14293 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4133
  total_bytes: 4133
}

2021-05-09 12:46:41,085 INFO    MainThread:14293 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4133
  total_bytes: 4133
}

2021-05-09 12:46:41,187 INFO    MainThread:14293 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4133
  total_bytes: 4133
}

2021-05-09 12:46:41,288 INFO    MainThread:14293 [wandb_run.py:_wait_for_finish():1646] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4133
  total_bytes: 4133
}

